android/modules/calendar/src/java/ti/modules/titanium/calendar/EventProxy.java

```diff
@@ -243,6 +244,36 @@ public static EventProxy createEvent(CalendarProxy calendar, KrollDict data)
 		}
 	}
 
+	private AttendeeProxy[] getAttendeeProxies() {
+		final String[] attendeeProjection = new String[]{
+				CalendarContract.Attendees._ID,
+				CalendarContract.Attendees.EVENT_ID,
+				CalendarContract.Attendees.ATTENDEE_NAME,
+				CalendarContract.Attendees.ATTENDEE_EMAIL,
+				CalendarContract.Attendees.ATTENDEE_TYPE,
+				CalendarContract.Attendees.ATTENDEE_RELATIONSHIP,
+				CalendarContract.Attendees.ATTENDEE_STATUS
+		};
+		final String query = "(" + CalendarContract.Attendees.EVENT_ID + " = ?)";
+		final String[] args = new String[]{id};
+		ContentResolver contentResolver = TiApplication.getInstance().getContentResolver();
+		final Cursor cursor = contentResolver.query(CalendarContract.Attendees.CONTENT_URI, attendeeProjection, query, args, null);
```

Comment:
The `ContentResolver.query()` method can return null. We need to do a null check on `cursor` before using it below.